e2e/bitsrc-tester.js

```diff
@@ -1,16 +1,18 @@
 import requestify from 'requestify';
 
 // const apiBaseUrl = process.env.NODE_ENV === 'production' ? 'https://api.bitsrc.io' : 'https://api-stg.bitsrc.io';
+const supportTestingOnBitsrc = false;
```

Comment:
I didn't like much the format we used for AppVeyor because of the extra indentation.
We can easily change my variable from
`const supportTestingOnBitsrc = false;`
to
`const supportTestingOnBitsrc = process.env.BITSRC_TESTING;`
if needed.